src/plugins/data/common/data_frames/utils.ts

```diff
@@ -290,6 +290,26 @@ export const getTimeField = (
     : fields.find((field) => field.type === 'date');
 };
 
+/**
+ * Parses timepicker datetimes using datemath package. Will attempt to parse strings such as
+ * "now - 15m"
+ *
+ * @param dateRange - of type TimeRange
+ * @returns object with fromDate and toDate, both of which will be in utc time and formatted to
+ * 'YYYY-MM-DD HH:mm:ss.SSS'
+ */
+export const formatTimePickerDate = (dateRange: TimeRange) => {
+  const dateMathParse = (date: string) => {
+    const parsedDate = datemath.parse(date);
+    return parsedDate ? parsedDate.utc().format('YYYY-MM-DD HH:mm:ss.SSS') : '';
```

Comment:
do we think we should make the format configurable by the plugin that interacts with this?

or should we default to pulling from [advanced settings](https://playground.opensearch.org/app/management/opensearch-dashboards/settings) to get the date format?

Reply:
Making it configurable by the plugin would be better actually, since we're not displaying this date format anywhere and currently its used to send to the sql plugin. I'll change it to have it be a parameter in the function.